src/main/java/org/broadinstitute/consent/http/service/EmailService.java

```diff
@@ -233,6 +245,54 @@ private void sendNewDARRequestEmail(
         user.getUserId());
   }
 
+  public void sendExpirationNotices() {
+    sendDARExpirationReminderNotices();
+    sendDARExpirationNotices();
+  }
+
+  private void sendDARExpirationNotices() {
+    EmailType emailType = EmailType.DAR_EXPIRED;
+    String interval = "1 year";
+    sendDARMessageToList(emailType, interval);
+  }
+
+  private void sendDARExpirationReminderNotices() {
+    EmailType emailType = EmailType.DAR_EXPIRATION_REMINDER;
+    String interval = "11 months";
+    sendDARMessageToList(emailType, interval);
+  }
+
+  private void sendDARMessageToList(EmailType type, String interval) {
+    List<DataAccessRequest> expiredDars = dataAccessRequestDAO.findAgedDARsByEmailTypeOlderThanInterval(
+        type.getTypeInt(), interval, MINIMUM_SUBMITTED_DATE_FOR_DAR_EXPIRATIONS);
+    expiredDars.forEach(expiredDar -> {
+      try {
+        String referenceId = expiredDar.getReferenceId();
+        User user = userDAO.findUserById(expiredDar.getUserId());
+        String darCode = expiredDar.getDarCode();
+        String userName = user.getDisplayName();
+        if (user.getEmail() == null) {
+          // Do not throw here.  Log information about the DAR since this will continue
+          // to appear broken until manual intervention is taken to resolve the missing user
+          // email address
+          logWarn(String.format(
```

Comment:
I think it might be nice to log this to Sentry as well so we can be notified via Slack. We don't have a warn level sentry logger, but I think this is an example where we would need that. Potential implementation:
```
  default void logSentryWarning(String message) {
    logWarn(message);
    SentryEvent event = new SentryEvent();
    event.setLevel(SentryLevel.WARNING);
    Message sentryMessage = new Message();
    sentryMessage.setMessage(message);
    event.setMessage(sentryMessage);
    Sentry.captureEvent(new SentryEvent());
  }
```

Reply:
Would making this a logError event satisfy that need just as well?

Reply:
We don't have a `logError` currently, but if we did and it did something similar, then yes.